### ISSUE TITLE
Crash on login: MissingSchema Invalid URL 'Noneusers/auth0|581271043ac3966f6e37fdad'

### ISSUE THREAD

**Issue by Fak3** [frictionlessdata/dpr-api#22]
When i signup with email and password on auth0 i get this error from dpr-api server:

```
127.0.0.1 - - [28/Oct/2016 00:26:03] "GET /api/login HTTP/1.1" 302 -
127.0.0.1 - - [28/Oct/2016 00:26:30] "GET /api/callback?code=pAdGIKIOBLnwD9if HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/u1/.virtualenvs/dpr/lib/python2.7/site-packages/flask/app.py", line 2000, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/u1/.virtualenvs/dpr/lib/python2.7/site-packages/flask/app.py", line 1991, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/u1/.virtualenvs/dpr/lib/python2.7/site-packages/flask/app.py", line 1567, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/u1/.virtualenvs/dpr/lib/python2.7/site-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/u1/.virtualenvs/dpr/lib/python2.7/site-packages/flask/app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/u1/.virtualenvs/dpr/lib/python2.7/site-packages/flask/app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/u1/.virtualenvs/dpr/lib/python2.7/site-packages/flask/app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/u1/.virtualenvs/dpr/lib/python2.7/site-packages/flask/app.py", line 1625, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/z/pproj/dpr-api/app/mod_api/controllers.py", line 196, in callback_handling
    update_user_secret(user_id)
  File "/home/z/pproj/dpr-api/app/utils/auth0_helper.py", line 47, in update_user_secret
    response = requests.request("PATCH", url, data=json.dumps(payload), headers=headers)
  File "/home/u1/.virtualenvs/dpr/lib/python2.7/site-packages/requests/api.py", line 56, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/u1/.virtualenvs/dpr/lib/python2.7/site-packages/requests/sessions.py", line 461, in request
    prep = self.prepare_request(req)
  File "/home/u1/.virtualenvs/dpr/lib/python2.7/site-packages/requests/sessions.py", line 394, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/home/u1/.virtualenvs/dpr/lib/python2.7/site-packages/requests/models.py", line 294, in prepare
    self.prepare_url(url, params)
  File "/home/u1/.virtualenvs/dpr/lib/python2.7/site-packages/requests/models.py", line 354, in prepare_url
    raise MissingSchema(error)
MissingSchema: Invalid URL 'Noneusers/auth0|581271043ac3966f6e37fdad': No schema supplied. Perhaps you meant http://Noneusers/auth0|581271043ac3966f6e37fdad?
```

**Comment by Fak3** [comment]
This happens when AUTH0_API_AUDIENCE is not set in the environment. Related: #25

**Comment by subhankarb** [comment]
Resolved #40
